README.md

```diff
@@ -371,6 +371,52 @@ impl<T> Stack<T> {
   assert_eq!(B::foo(1), 2);
   ```
 
+- Delegate associated constants
+
+```rs
+use delegate::delegate;
+
+trait WithConst {
```

Comment:
Can you please add this bullet point also to `src/lib.rs`? There is a doc comment at the top of the file, so that it renders in docs.rs. The duplication is a bit unfortunate, but it is what it is.

Reply:
oh right ! I forgot it !
Speaking of duplication, I saw some [projects](https://github.com/embassy-rs/embassy/blob/main/embassy-boot-rp/src/lib.rs) do 
`#![doc = include_str!("../README.md")]`

This could be a nice way to avoid duplication 🤔

Reply:
Indeed, but I don't want to include the whole README, just the code examples, and that would be annoying to set up.